device/transport/amqp/devdoc/amqp_device_method_client_requirements.md

```diff
@@ -44,14 +48,40 @@ client.sendMethodResponse(methodResponse, function(err, result) {
 
 **SRS_NODE_AMQP_DEVICE_METHOD_CLIENT_16_003: [** The `AmqpDeviceMethodClient` shall inherit from the `EventEmitter` class. **]**
 
+### attach(callback)
+
+**SRS_NODE_AMQP_DEVICE_METHOD_CLIENT_16_014: [** The `AmqpDeviceMethodClient` object shall set 2 properties of any AMQP link that it create:
+- `com.microsoft:api-version` shall be set to the current API version in use.
+- `com.microsoft:channel-correlation-id` shall be set to the identifier of the device (also often referred to as `deviceId`). **]**
```

Comment:
>channel-correlation-id [](start = 17, length = 22)

This must be a service requirement.  Hard to imagine that there isn't some other property of the link that contains the deviceId.  Not pointing out a problem here, it just seems odd.